int32 none
---
string[] topic_name
string[] topic_type
int32[] topic_freq